extensions/ql-vscode/src/cli.ts

```diff
@@ -509,12 +509,12 @@ export class CodeQLCliServer implements Disposable {
     testPaths: string[], workspaces: string[], options: TestRunOptions
   ): AsyncGenerator<TestCompleted, void, unknown> {
 
-    const subcommandArgs = [
+    const subcommandArgs = this.cliConfig.additionalTestArguments.toString().split(/\s+/).concat([
```

Comment:
Comment from @henrymercer. This will fail things like `--string-arg "hello world"`. We will need some more sophisticated parsing here.

Reply:
You could change the type of the setting to `array` and each arg is one string element of the array, but then they can only be editted in the settings.json see: https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema

This may not be so bad as long as you have a proper description in the text.